src/strategies/safe-vested/index.ts

```diff
@@ -1,4 +1,3 @@
-import fetch from 'cross-fetch';
```

Comment:
But we use fetch here https://github.com/snapshot-labs/snapshot-strategies/blob/eca18b39bf5093087ee6fdf98a6349dc882fdb8b/src/strategies/safe-vested/index.ts#L45

Reply:
i will update it to use customFetch, but JIC if not needed LMK